diff -uNr openmediavault-ldap/usr/share/openmediavault/scripts/ldap openmediavault-ldap.patched/usr/share/openmediavault/scripts/ldap
--- openmediavault-ldap/usr/share/openmediavault/scripts/ldap	2012-10-09 20:13:10.000014701 +0200
+++ openmediavault-ldap.patched/usr/share/openmediavault/scripts/ldap	2012-10-09 20:34:46.532017324 +0200
@@ -38,18 +38,17 @@
 OMV_LDAP_CONFIG=${OMV_LDAP_CONFIG:-"/etc/ldap/ldap.conf"}
 OMV_LDAP_LDAPVERSION=${OMV_LDAP_LDAPVERSION:-"3"}
 OMV_LDAP_LIBNSS_CONFIG=${OMV_LDAP_LIBNSS_CONFIG:-"/etc/libnss-ldap.conf"}
-OMV_LDAP_LIBNSS_SECRET=${OMV_LDAP_LIBNSS_SECRET:-"/etc/libnss-ldap.secret"}
+OMV_LDAP_LIBNSS_SECRET=${OMV_LDAP_LIBNSS_SECRET:-"/etc/ldap.secret"}
 OMV_LDAP_LIBNSS_BINDPOLICY=${OMV_LDAP_LIBNSS_BINDPOLICY:-"soft"}
-OMV_LDAP_PAM_CONFIG=${OMV_LDAP_PAM_CONFIG:-"/etc/pam_ldap.conf"}
+OMV_LDAP_PAM_CONFIG=${OMV_LDAP_PAM_CONFIG:-"/etc/ldap.conf"}
 
 xmlstarlet sel -t \
   -m "//services/ldap" \
 	-v "concat('base ',base)" -n \
 	-v "concat('uri ldap://',host)" -n \
 	-o "ldap_version ${OMV_LDAP_LDAPVERSION}" -n \
-	-v "concat('binddn ',rootbinddn)" -n \
-	-v "concat('bindpw ',rootbindpw)" -n \
-	-o "pam_password crypt" -n \
+	-v "concat('rootbinddn ',rootbinddn)" -n \
+	-o "pam_password md5" -n \
   -b \
   ${OMV_CONFIG_FILE} | xmlstarlet unesc > ${OMV_LDAP_PAM_CONFIG}
 
@@ -61,19 +60,19 @@
   -b \
   ${OMV_CONFIG_FILE} | xmlstarlet unesc > ${OMV_LDAP_CONFIG}
 
-xmlstarlet sel -t \
-  -m "//services/ldap" \
-	-v "concat('base ',base)" -n \
-	-v "concat('uri ldap://',host)" -n \
-	-o "ldap_version ${OMV_LDAP_LDAPVERSION}" -n \
-	-v "concat('rootbinddn ',rootbinddn)" -n \
-	-o "bind_policy ${OMV_LDAP_LIBNSS_BINDPOLICY}" -n \
-	-v "concat('nss_base_passwd ',usersuffix,',',base)" -n \
-	-v "concat('nss_base_shadow ',usersuffix,',',base)" -n \
-	-v "concat('nss_base_group ',groupsuffix,',',base)" -n \
-	-i "string-length(extraoptions) > 0" -v "extraoptions" -n -b \
-  -b \
-  ${OMV_CONFIG_FILE} | xmlstarlet unesc > ${OMV_LDAP_LIBNSS_CONFIG}
+# xmlstarlet sel -t \
+#  -m "//services/ldap" \
+#	-v "concat('base ',base)" -n \
+#	-v "concat('uri ldap://',host)" -n \
+#	-o "ldap_version ${OMV_LDAP_LDAPVERSION}" -n \
+#	-v "concat('rootbinddn ',rootbinddn)" -n \
+#	-o "bind_policy ${OMV_LDAP_LIBNSS_BINDPOLICY}" -n \
+#	-v "concat('nss_base_passwd ',usersuffix,',',base)" -n \
+#	-v "concat('nss_base_shadow ',usersuffix,',',base)" -n \
+#	-v "concat('nss_base_group ',groupsuffix,',',base)" -n \
+#	-i "string-length(extraoptions) > 0" -v "extraoptions" -n -b \
+#  -b \
+#  ${OMV_CONFIG_FILE} | xmlstarlet unesc > ${OMV_LDAP_LIBNSS_CONFIG}
 
 xmlstarlet sel -t \
   -m "//services/ldap" \
